magda-web-client/src/reducers/datasetSearchReducer.js

```diff
@@ -46,18 +46,31 @@ const datasetSearchReducer = (
 ) => {
     switch (action.type) {
         case "REQUEST_RESULTS":
+            let queryObject = action.queryObject;
+            console.log(action.apiQuery);
```

Comment:
Need to remove this before merging :)

Reply:
Yea i realised that when you select a single publisher, there is no array of publishers, just a string. And in FacetHeader.js it is rendering activeOptions.length. And in the case where there is 1 publisher selected it is just rendering the length of text. So when you're selecting Geoscience Australia it is rendering 48 as in 48 characters of string 😅 .